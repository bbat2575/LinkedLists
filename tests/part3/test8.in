NEW 0
NEW 0
NEW 0
NEW 3
{0}
{1}
{2}
NEW 1
cowabunga dude!
INSERT 0 0 {4}
DELETE 3 0
INSERT 0 0 {4}
INSERT 2 0 {1}
INSERT 2 0 {4}
DELETE 3 -1
DELETE 3 0
INSERT 2 0 {1}
INSERT 2 0 {4}
VIEW-NESTED 2
TYPE 2
