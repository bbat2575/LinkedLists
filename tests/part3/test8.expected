
Type HELP for a list of options
Press <CTRL + D> to Quit

> List 0: 

> List 1: 

> List 2: 

> Nested 3: {List 0} -> {List 1} -> {List 2}

> List 4: cowabunga dude!

> INVALID COMMAND: INSERT

> List 3: {List 1} -> {List 2}

> Nested 0: {List 4}

> INVALID COMMAND: INSERT

> INVALID COMMAND: INSERT

> List 3: {List 1}

> List 3: 

> Nested 2: {List 1}

> Nested 2: {List 4} -> {List 1}

> {cowabunga dude!} -> {}

> reference -> reference

> 
